### PR TITLE
Fgemv with par seq helper

### PR DESCRIPTION
**Pull request by ZHG2017**
Modified the fgemv interface to use ParSeqHelper for later use in SAGEMATHS then unified pfgemv and fgemv both to fgemv but with different templated helper which might be more consistent for the interface

### CODE REVIEW

File: fflas-ffpack/fflas/fflas_pfgemv.inl [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+           const size_t m,
+           const size_t n,
+           const typename Field::Element alpha,
+           const typename Field::ConstElement_ptr A, const size_t lda,
+           const typename Field::ConstElement_ptr X, const size_t incX,
+           const typename Field::Element beta,
+           typename Field::Element_ptr Y, const size_t incY,
+           ParSeqHelper::Sequential seqH ){
+
+        return fgemv(F, ta, m, n, alpha, A, lda, X, incX, beta, Y, incY);
```
Comment (ClementPernet) [inline fflas-ffpack/fflas/fflas_pfgemv.inl]: and also change the call to fgemv without helper in one with a default MMHelper.

Comment (ClementPernet) [reply fflas-ffpack/fflas/fflas_pfgemv.inl]: Still unresolved

Comment (ZHG2017) [reply fflas-ffpack/fflas/fflas_pfgemv.inl]: I think this is a file only exists on the remote branch, I have locally deleted this file.

File: fflas-ffpack/fflas/fflas_fgemv.inl [hunk 2]
```diff
@@ -463,6 +463,39 @@ namespace FFLAS{
...
+           const size_t m,
+           const size_t n,
+           const typename Field::Element alpha,
+           const typename Field::ConstElement_ptr A, const size_t lda,
+           const typename Field::ConstElement_ptr X, const size_t incX,
+           const typename Field::Element beta,
+           typename Field::Element_ptr Y, const size_t incY,
+           ParSeqHelper::Sequential seqH ){
+        MMHelper<Field, MMHelperAlgo::Classic, typename FFLAS::ModeTraits<Field>::value, ParSeqHelper::Sequential > pH (F,m,n,1,seqH);
+        return fgemv(F, ta, m, n, alpha, A, lda, X, incX, beta, Y, incY);
```
Comment (ClementPernet) [inline fflas-ffpack/fflas/fflas_fgemv.inl]: Sure but the problem is still here: you build pH but never use it.

Comment (ZHG2017) [reply fflas-ffpack/fflas/fflas_fgemv.inl]: OK, I see, I just did a instant push, I forgot about using this in the function. I will correct this shortly.

**Comment by ClementPernet** [comment]
LGTM

**Pull request merged**